src/onnx/onnx.cpp

```diff
@@ -1426,17 +1426,14 @@ struct onnx_parser
         for(auto&& f : graph.initializer())
         {
             initializer_data[f.name()] = f;
```

Comment:
Do we even need the `initializer_data` map now? We can just create the literals instead.

Reply:
removed it.